internal/exporter/prometheus/collector/sysfs.go

```diff
@@ -0,0 +1,24 @@
+package collector
+
+import "github.com/prometheus/procfs/sysfs"
+
+// sysFS is an interface to prometheus/procfs/sysfs
+type sysFS interface {
+	Zones() ([]sysfs.RaplZone, error)
+}
+
+type realSysFS struct {
+	sysfs sysfs.FS
+}
+
+func (s *realSysFS) Zones() ([]sysfs.RaplZone, error) {
```

Comment:
we should reuse cpuPowerMeter here since the raplReader removes non-standard rapl zones ( a functionality that we want to preserve here as well). Also when we implement filtering of zones, we should only show the zones that are filtered. 

We could instead have an interface that raplPowerMeter satisfies which returns only the EnergyZones() which provides all information.

Reply:
then we can have this metric as part of power collecor itself.

Reply:
Indeed, didn't think of that :)